.github/workflows/build-and-runtime-test.yml

```diff
@@ -148,9 +148,8 @@ jobs:
         with:
           metric-name: BuildAndRuntimeTestFailure
           value: 1
-          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_METRIC_LOGGER }}
-          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_METRIC_LOGGER }}
-          AWS_REGION: us-east-2
+          role-to-assume: ${{ secrets.METRIC_LOGGER_ROLE_ARN}}
+          aws-region: us-east-2
```

Comment:
nit: if we only use `us-east-2`, i wonder if we can save it as a workflow environment variable

Reply:
I like the idea. Let's scope this to a different task though, since we have `us-east-2` hard coded in other places too and we can do 1 PR to update them.